utils/crawler.py

```diff
@@ -7,11 +7,13 @@
 from time import sleep
 import xml.etree.ElementTree as ET
```

Comment:
Maybe optimize the imports by removing this as I cannot see it has been used anywhere in this file.